lib/chef/knife/windows_helper.rb

```diff
@@ -27,7 +28,8 @@ class WindowsHelper < Knife
 
       banner "#{BootstrapWindowsWinrm.banner}\n" +
```

Comment:
It can probably just go away, since it was more a rollup of "all" the knife-windows options under a Windows header.  It doesn't have the cert commands listed at all either.